lib/MusicBrainz/Server/Edit/Relationship/RelatedEntities.pm

```diff
@@ -17,15 +18,25 @@ around _build_related_entities => sub {
         my $model = type_to_model($type);
         my @ids = @{ $direct->{$type} };
         my @entities = values %{ $self->c->model($model)->get_by_any_ids(@ids) };
-        $self->c->model('ArtistCredit')->load(@entities);
-        $direct->{artist} ||= [];
-        push @{ $direct->{artist} }, map { $_->artist_id }
-            map { $_->artist_credit->all_names }
-                @entities;
+        if ($type ~~ entities_with('artist_credits')) {
```

Comment:
This doesn't seem to work. I'd honestly recommend to avoid ever using `~~`, since it's experimental and has complicated semantics. :( But luckily we don't need anything fancy for this check:
```suggestion
        if ($ENTITIES{$type}{artist_credits}) {
```